### DISCUSSION TITLE
Could not establish a connection to the machine running this server. Please try again.

### DISCUSSION THREAD

**Discussion by TardisKing40** [vilhelmprytz/pterodactyl-installer#302] [Q&A]
![Capture](https://user-images.githubusercontent.com/38845418/156845199-f9524265-0af4-40d3-abf4-9e18cde0a213.PNG)

I setup everything correctly, I have no idea why this keeps happening.
I have tried fully resetting the server and reinstalling it, it seems to do nothing

**Comment by TardisKing40** [comment]
After having a look at the nodes, My main node seems to be offline.
![Capture](https://user-images.githubusercontent.com/38845418/156845904-073fd135-eff5-49d7-a256-e5fda2b39b4c.PNG)

After typing in the configuration startup command I get this error.
![Capture](https://user-images.githubusercontent.com/38845418/156846272-e5ab2e19-5bd8-4758-83fc-640b4d34a264.PNG)


Anyone know why this is?

**Comment by vilhelmprytz** [comment]
This is likely not a problem with the script, so I will convert this into a discussion thread :slightly_smiling_face:

**Comment by vilhelmprytz** [comment]
Are you using the automatic Let's Encrypt certificate feature of the script?

**Reply by TardisKing40** [reply]
yes

**Comment by ogresosa** [comment]
im having the same issue can someone help
